config/dev-active/views.view.modules.json

```diff
@@ -455,7 +455,7 @@
                             "description": "",
                             "use_operator": 0,
                             "operator": "combine_op",
-                            "identifier": "combine",
+                            "identifier": "s",
```

Comment:
Can we use an identifier that's a little more descriptive than 's'?  How about 'search'? :)